packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -1,171 +1,183 @@
+                    :value="consumer.postcode"
+                    maxlength="50"
+                    :label-text="$t('consumer.postcodeLabel')"
+                    :placeholder="$t('consumer.postcodePlaceholder')"
+                    @blur="onBlur('postcode')"
+                    @input="onEditConsumer('postcode', $event)">
+                    <template
+                        v-if="$v.consumer.postcode.$invalid"
+                        #error>
+                        <f-error-message v-show="!$v.consumer.postcode.required && $v.consumer.postcode.$dirty">
+                            {{ $t('validation.postcodeRequired') }}
+                        </f-error-message>
+                        <f-error-message v-show="!$v.consumer.postcode.isValidPostcode && $v.consumer.postcode.required">
+                            {{ $t('validation.postcodeInvalid') }}
+                        </f-error-message>
+                    </template>
+                </form-field>
+
+                <f-button
+                    :class="[$style['c-accountInfo-submitButton']]"
+                    data-test-id="account-info-submit-button"
+                    button-type="primary"
+                    button-size="large"
+                    is-full-width
+                    action-type="submit"
+                    :is-loading="isFormSubmitting">
+                    {{ $t('buttons.saveChanges') }}
+                </f-button>
+            </form>
 
             <f-button
-                :class="[$style['c-accountInfo-submitButton']]"
+                :class="[$style['c-accountInfo-changePasswordButton']]"
                 data-test-id="account-info-submit-button"
-                button-type="primary"
+                button-type="secondary"
+                href="/change-password?returnurl=/account/info"
                 button-size="large"
                 is-full-width
-                action-type="submit"
-                :is-loading="isFormSubmitting">
-                {{ $t('buttons.saveChanges') }}
+                action-type="submit">
+                {{ $t('buttons.changePassword') }}
             </f-button>
-        </form>
-
-        <f-button
-            :class="[$style['c-accountInfo-changePasswordButton']]"
-            data-test-id="account-info-submit-button"
-            button-type="secondary"
-            href="/change-password?returnurl=/account/info"
-            button-size="large"
-            is-full-width
-            action-type="submit">
-            {{ $t('buttons.changePassword') }}
-        </f-button>
-
-        <delete-account />
-    </f-card>
+
+            <delete-account />
+        </f-card>
+        <f-card-with-content
+            v-else
+            data-test-id="account-info-error-card"
+            :card-heading="$t('errorMessages.errorHeading')"
+            :card-description="$t(error.messageKey)">
+            <template #icon>
+                <bag-sad-bg-icon />
+            </template>
```

Comment:
Do we want a button linking back to the homepage or something? Happy for it to be added later if we don't really know yet

Reply:
Yes, I am going to raise a question with design and address in a later PR. The design shows a 'Go back' button but we need to understand exactly how this should function.